--- vmmon-only/linux/vmmonInt.h	2014-06-13 02:38:24.000000000 +0200
+++ vmmon-only/linux/vmmonInt.h	2014-11-05 14:58:20.712535118 +0100
@@ -28,11 +28,7 @@
  * Hide all kernel compatibility stuff in these macros and functions.
  */
 
-#ifdef VMW_HAVE_SMP_CALL_3ARG
 #define compat_smp_call_function(fn, info, wait) smp_call_function(fn, info, wait)
-#else
-#define compat_smp_call_function(fn, info, wait) smp_call_function(fn, info, 1, wait)
-#endif
 
 /*
  * Although this is not really related to kernel-compatibility, I put this
