--- Quazaa/3rdparty/communi-desktop/src/soundnotification.cpp	2014-08-22 11:30:37.522995622 +0200
+++ Quazaa/3rdparty/communi-desktop/src/soundnotification.cpp	2014-08-22 11:42:39.174690710 +0200
@@ -15,9 +15,14 @@
 #include "soundnotification.h"
 #include "singleapplication.h"
 
-#include <QStandardPaths>
 #include <QDir>
 
+#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
+#include <QDesktopServices>					
+#else
+#include <QStandardPaths>
+#endif
+
 #if defined(QT_MULTIMEDIA_LIB)
 #include <QMediaPlayer>
 #elif defined(QT_PHONON_LIB)
