--- Quazaa/3rdparty/communi-desktop/src/gui/messageview.cpp	2014-08-22 12:13:33.162390018 +0200
+++ Quazaa/3rdparty/communi-desktop/src/gui/messageview.cpp	2014-08-22 12:17:35.593632689 +0200
@@ -285,9 +285,17 @@ void MessageView::showHelp(const QString
 void MessageView::sendMessage(const QString& line)
 {
     if (d.messageTimer->isActive()) {
+#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
+		d.messages.append(line.split(QRegExp("[\\r\\n]"), QString::KeepEmptyParts));
+#else
         d.messages.append(line.split(QRegularExpression("[\\r\\n]"), QString::KeepEmptyParts));
+#endif
     } else {
+#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
+		d.messages = line.split(QRegExp("[\\r\\n]"), QString::KeepEmptyParts);
+#else
         d.messages = line.split(QRegularExpression("[\\r\\n]"), QString::KeepEmptyParts);
+#endif
         sendMessageLine();
         d.messageTimer->start(1000);
     }
