--- Quazaa/3rdparty/communi-desktop/src/gui/messageview.cpp	2014-08-22 11:58:41.619226498 +0200
+++ Quazaa/3rdparty/communi-desktop/src/gui/messageview.cpp	2014-08-22 12:03:09.576242971 +0200
@@ -25,7 +25,11 @@
 #include "quazaasysinfo.h"
 #include "quazaaglobals.h"
 #include "chatconverter.h"
+#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
+#include <QRegExp>					
+#else
 #include <QRegularExpression>
+#endif
 #include <QAbstractTextDocumentLayout>
 #include <QDesktopServices>
 #include <QTextBlock>
