--- Quazaa/Quazaa.pro	2014-09-15 09:01:58.631154467 +0200
+++ Quazaa/Quazaa.pro	2014-09-15 09:08:21.359071029 +0200
@@ -100,7 +100,7 @@ QMAKE_CLEAN += $$VERSION_HEADER_PATH
 # Language stuff
 isEmpty(QMAKE_LRELEASE) {
 		win32:QMAKE_LRELEASE = $$[QT_INSTALL_BINS]\\lrelease.exe
-		else:QMAKE_LRELEASE = $$[QT_INSTALL_BINS]/lrelease
+		else:QMAKE_LRELEASE = /usr/bin/lrelease
 }
 
 updateqm.input = TRANSLATIONS
